### PR TITLE
DataFrameClient for InfluxDB 0.9

### PR DESCRIPTION
**Pull request by tzonghao**
#108 

* `get_list_series` returns a mapping from measurement names to DataFrame representing tags for the named measurement.
* `query` returns a mapping from (measurement, (tag (k, v) pairs), to DataFrame representing time series for the measurement (qualified by tags) in question. Query without tags should simply use measurement as keys.
* `write_points` converts a mapping of DataFrame objects as `points`, then hand them over to super's `write_points`.

### CODE REVIEW

**Comment by ReAzem** [comment]
Hi,

Thank you for taking on this issue :)

Have you looked at ``client_test_with_server.py``? I would like some DaframeClient tests there.

You are using the ``u'string'`` syntax, which does not work in python 3.2.

Other than that, this looks great!

//cc @timtroendle

**Comment by tzonghao** [comment]
@ReAzem In case you haven't reviewed it, a few basic tests were added in `client_test_with_server.py` in this pull request as suggested. The 3.2 syntax problem is also fixed. I've also changed the key data structure for data dictionary's keys, for both querying and submitting, where strings can be used as keys if no tags are used. Otherwise, the keys can be a tuple of (measurement, sorted tuple of tag key-value pairs).

**Comment by ReAzem** [comment]
@tzonghao Hello,

Than you for the great work. Why are you not using the ResultSet Object? You should not have to use the raw property.

**Comment by tzonghao** [comment]
@ReAzem You're right. Using ResultSet is cleaner. It's converted in the last commit.

**Comment by ReAzem** [comment]
@tzonghao Can you change the write_points function to accept three arguments instead?

- measurment name
- tags (optional)
- dataframe

I think it would be more intuitive than a tuple of key-value tuples.

**Comment by tzonghao** [comment]
@ReAzem Sure, I can make that change. It also implicates that `write_points` will accept one DataFrame at a time, instead of multiple DataFrames.

**Comment by ReAzem** [comment]
@tzonghao  I think it would be worth it, do you agree?

**Comment by tzonghao** [comment]
I agree. It's a good decision.

**Comment by tzonghao** [comment]
@ReAzem It looks like an InfluxDB error in the last Travis run that failed the py34 test. Can you trigger a rerun?

File: examples/tutorial_pandas.py [hunk 1]
```diff
@@ -22,6 +22,9 @@ def main(host='localhost', port=8086):
     print("Write DataFrame")
     client.write_points({'demo': df})
 
+    print("Write DataFrame with Tags")
+    client.write_points({('demo', (('k1', 'v1'), ('k2', 'v2'))): df})
```
Comment (ReAzem) [inline examples/tutorial_pandas.py]: I don't think this works anymore

Comment (tzonghao) [inline examples/tutorial_pandas.py]: good catch

**Comment by ReAzem** [comment]
Great work! I will be merging this for now. I'm sure we can keep on improving it after feedback from other users.

**Pull request merged**